src/utils/helpers.js

```diff
@@ -0,0 +1,23 @@
+export function renderLoading(isLoading, button, defaultText='Save' , loadingText="Saving. . .") {
+  if (isLoading) {
+    button.textContent = loadingText;
+  } else {
+    button.textContent = defaultText;
+  }
+}
+
+export function handleSubmit(request, evt, loadingText = 'Saving...') {
+  evt.preventDefault();
+
+  const submitButton = evt.submitter;
+  const initialText = submitButton.textContent;
+  renderLoading(true, submitButton, initialText, loadingText);
+  request()
+    .then(() => {
+      // evt.target.reset();
```

Comment:
Please, uncomment it because it's needed

* You should **clear the inputs** after a successful adding of a new card to let the user add the 2nd one again without having to remove the old data manually.